changelogs/fragments/78678-add-a-g-install-offline.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - ``ansible-galaxy collection install`` - add an ``--offline`` option to prevent querying distribution servers (https://github.com//issues/77443).
```

Looks like this accidentally got dropped in my previous suggestion:
```suggestion
  - ``ansible-galaxy collection install`` - add an ``--offline`` option to prevent querying distribution servers (https://github.com/ansible/ansible/issues/77443).
```

---

Fixed.